offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsView.groovy

```diff
@@ -143,6 +143,7 @@ class MaintainProductsView extends VerticalLayout{
         productGrid.addSelectionListener({
             if(it.firstSelectedItem.isPresent()){
                 updateProductDescription(it.firstSelectedItem.get())
+                viewModel.selectedProduct = it.firstSelectedItem.get()
```

Comment:
NIT: I think a dedicated variable makes the code easier to read here
```suggestion
            def selectedProduct = it.firstSelectedItem
            if(selectedProduct.isPresent()){
                updateProductDescription(selectedProduct.get())
                viewModel.selectedProduct = selectedProduct.get()
```

Reply:
should be in the isPresent block

Reply:
?